lib/parsers/lock-parser-base.ts

```diff
@@ -101,7 +104,10 @@ export abstract class LockParserBase implements LockfileParser {
     const depMap: DepMap = this.getDepMap(yarnLock, manifestFile.resolutions);
 
     // all paths are identified, we can create a graph representing what depends on what
-    const depGraph: graphlib.Graph = this.createGraphOfDependencies(depMap);
+    const depGraph: graphlib.Graph = this.createGraphOfDependencies(
+      depMap,
+      strict,
```

Comment:
Can we be more explicit with the naming? "strictOutOfSync" so its easier to find and delete later.